src/main/java/core/basesyntax/figure/ColorSupplier.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.figure;
+
+import java.util.Random;
+
+public class ColorSupplier {
```

Comment:
Don't use static methods in your solution. However, in this case, since `getRandomColor()` does not rely on any instance state and purely provides a utility function, using a static method could be considered acceptable. It's a matter of context and design choice.